src/main/java/edu/hawaii/its/groupings/controller/HomeController.java

```diff
@@ -226,4 +225,7 @@ public String importErrorModal() {
     public String dynamicModal() {
         return "modal/dynamicModal";
     }
+
+    @GetMapping(value = "/modal/groupingOwnersModal")
+    public String groupingOwnersModal() {return "modal/groupingOwnersModal";}
```

Comment:
Could you open up this function so that it looks like this:
```
public String groupingOwnersModal() {
    return "modal/groupingOwnersModal";
}
```
IntelliJ compresses a function that is just a single return line, but once you click on the curly bracket, it will open up the function.